src/modules/SimpleSelect.jsx

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable react/prop-types */
+import React, { useEffect } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import { getInitialBoardArray } from './utils/helpers';
+
+const useStyles = makeStyles((theme) => ({
+  formControl: {
+    margin: theme.spacing(1),
+    minWidth: 120,
+    fontSize: '2rem',
+  },
+  label: {
+    fontSize: '2rem',
+  },
+  select: {
+    fontSize: '2rem',
+  },
+  menuItem: {
+    fontSize: '2rem',
+    backgroundColor: `#e0e0e0`,
+  },
+  selectEmpty: {
+    marginTop: theme.spacing(2),
+  },
+}));
+
+export default function SimpleSelect(props) {
+  const classes = useStyles();
+  const { boardSize, setBoardSize, setHistory, setIsPopUpActive } = props;
+
+  const newGame = (size) => {
+    setHistory([
+      {
+        board: getInitialBoardArray(size * size),
+        score: 0,
+        win: false,
+        gameOver: false,
+      },
+    ]);
+  };
+
+  useEffect(() => {
+    setIsPopUpActive(false);
+  }, [boardSize]);
+
+  const handleChange = (event) => {
```

Comment:
restructuring can be used here: `({target: {value} })`

Reply:
yep, haven't got used to it yet)